pkg/controller/controller_binding.go

```diff
@@ -1168,6 +1163,11 @@ func (c *controller) pollServiceBinding(binding *v1beta1.ServiceBinding) error {
 
 	glog.V(4).Infof(pcb.Message("Processing"))
 
+	binding, err := makeServiceBindingClone(binding)
```

Comment:
any reason to not do this everywhere in a followup? looks nice.

Will conflict and need to merge with https://github.com/kubernetes-incubator/service-catalog/pull/1544

Reply:
Do you mean having a helper function for the cloning of other resource types?

Reply:
I'm +1 on that for future PRs. Feels cleaner